docker/pg-s3-backup/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:buster-slim as build
+
+# Install aws cli
+RUN apt-get update \
+    && apt-get install -y curl unzip \
+    && curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```

Comment:
Is there any way to specify a version on this curl?

Reply:
Done.